<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CellmlAnnotationViewCellmlListWidget</class>
 <widget class="QWidget" name="CellmlAnnotationViewCellmlListWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="layout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
  </layout>
  <action name="actionExpandAll">
   <property name="icon">
    <iconset resource="../res/CellMLAnnotationView.qrc">
     <normaloff>:/oxygen/actions/list-add.png</normaloff>:/oxygen/actions/list-add.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Expand All</string>
   </property>
   <property name="statusTip">
    <string>Expand all the children nodes</string>
   </property>
  </action>
  <action name="actionCollapseAll">
   <property name="icon">
    <iconset resource="../res/CellMLAnnotationView.qrc">
     <normaloff>:/oxygen/actions/list-remove.png</normaloff>:/oxygen/actions/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>&amp;Collapse All</string>
   </property>
   <property name="statusTip">
    <string>Collapse all the children nodes</string>
   </property>
  </action>
  <action name="actionRemoveCurrentMetadata">
   <property name="text">
    <string>&amp;Remove Current Metadata</string>
   </property>
   <property name="statusTip">
    <string>Remove the metadata associated with the current node</string>
   </property>
  </action>
  <action name="actionRemoveAllMetadata">
   <property name="text">
    <string>Remove &amp;All Metadata</string>
   </property>
   <property name="statusTip">
    <string>Remove all the metadata associated with the different nodes</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../res/CellMLAnnotationView.qrc"/>
 </resources>
 <connections/>
</ui>
